spec/controllers/gsis_controller_spec.rb

```diff
@@ -64,13 +65,26 @@ def delete_gsi
         gsi.save!
       end
 
-      it 'destroys the GSI' do
-        expect { delete_gsi }.to change(User, :count).by(-1)
-      end
-
       it "destroys the GSI's appointment" do
         expect { delete_gsi }.to change(Employment, :count).by(-1)
       end
+
+      describe 'if the GSI has other appointments' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.